src/renderer/root/components/App/Toasts/Toast/Toast.scss

```diff
@@ -2,7 +2,7 @@
   @include fade-in;
   display: flex;
   align-items: center;
-  max-width: 350px;
+  width: 350px;
```

Comment:
How is long text handled?
I like the fact that it's a fixed width though

Reply:
It wraps and aligns everything vertically.